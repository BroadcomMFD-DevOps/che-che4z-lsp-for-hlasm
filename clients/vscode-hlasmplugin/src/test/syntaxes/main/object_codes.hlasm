         CSECT

         MACRO
         MAC   &A
         L &A,0
         LA 0,&A
         MEND

         MAC 1
         MAC 2
         MAC 3
         MAC 4
         MAC 5

         END
