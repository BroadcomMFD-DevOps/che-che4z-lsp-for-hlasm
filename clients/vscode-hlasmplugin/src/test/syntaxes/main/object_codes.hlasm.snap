>         CSECT
#^^^^^^^^^ source.hlasm
#         ^^^^^ source.hlasm entity.name.function.hlasm
>
>         MACRO
#^^^^^^^^^ source.hlasm
#         ^^^^^ source.hlasm entity.name.function.hlasm
>         MAC   &A,&LBLNAME
#^^^^^^^^^ source.hlasm
#         ^^^ source.hlasm entity.name.function.hlasm
#            ^^^ source.hlasm
#               ^^ source.hlasm punctuation.definition.tag.hlasm variable.other.readwrite.hlasm
#                 ^ source.hlasm
#                  ^^^^^^^^ source.hlasm punctuation.definition.tag.hlasm variable.other.readwrite.hlasm
>         PR
#^^^^^^^^^ source.hlasm
#         ^^ source.hlasm entity.name.function.hlasm
>&LBLNAME LA 0,&A
#^^^^^^^^ source.hlasm support.class.hlasm variable.other.readwrite.hlasm
#        ^ source.hlasm
#         ^^ source.hlasm entity.name.function.hlasm
#           ^ source.hlasm
#            ^ source.hlasm punctuation.definition.tag.hlasm constant.numeric.hlasm
#             ^ source.hlasm
#              ^^ source.hlasm punctuation.definition.tag.hlasm variable.other.readwrite.hlasm
>         BPRP  0,&LBLNAME,&LBLNAME
#^^^^^^^^^ source.hlasm
#         ^^^^ source.hlasm entity.name.function.hlasm
#             ^^ source.hlasm
#               ^ source.hlasm punctuation.definition.tag.hlasm constant.numeric.hlasm
#                ^ source.hlasm
#                 ^^^^^^^^ source.hlasm punctuation.definition.tag.hlasm variable.other.readwrite.hlasm
#                         ^ source.hlasm
#                          ^^^^^^^^ source.hlasm punctuation.definition.tag.hlasm variable.other.readwrite.hlasm
>         MEND
#^^^^^^^^^ source.hlasm
#         ^^^^ source.hlasm entity.name.function.hlasm
>
>         MAC 1,LBL1
#^^^^^^^^^ source.hlasm
#         ^^^ source.hlasm entity.name.function.hlasm
#            ^ source.hlasm
#             ^ source.hlasm punctuation.definition.tag.hlasm constant.numeric.hlasm
#              ^ source.hlasm
#               ^^^^ source.hlasm punctuation.definition.tag.hlasm
>         MAC 2,LBL2
#^^^^^^^^^ source.hlasm
#         ^^^ source.hlasm entity.name.function.hlasm
#            ^ source.hlasm
#             ^ source.hlasm punctuation.definition.tag.hlasm constant.numeric.hlasm
#              ^ source.hlasm
#               ^^^^ source.hlasm punctuation.definition.tag.hlasm
>         MAC 3,LBL3
#^^^^^^^^^ source.hlasm
#         ^^^ source.hlasm entity.name.function.hlasm
#            ^ source.hlasm
#             ^ source.hlasm punctuation.definition.tag.hlasm constant.numeric.hlasm
#              ^ source.hlasm
#               ^^^^ source.hlasm punctuation.definition.tag.hlasm
>         MAC 4,LBL4
#^^^^^^^^^ source.hlasm
#         ^^^ source.hlasm entity.name.function.hlasm
#            ^ source.hlasm
#             ^ source.hlasm punctuation.definition.tag.hlasm constant.numeric.hlasm
#              ^ source.hlasm
#               ^^^^ source.hlasm punctuation.definition.tag.hlasm
>         MAC 5,LBL5
#^^^^^^^^^ source.hlasm
#         ^^^ source.hlasm entity.name.function.hlasm
#            ^ source.hlasm
#             ^ source.hlasm punctuation.definition.tag.hlasm constant.numeric.hlasm
#              ^ source.hlasm
#               ^^^^ source.hlasm punctuation.definition.tag.hlasm
>
>         END
#^^^^^^^^^ source.hlasm
#         ^^^ source.hlasm entity.name.function.hlasm
>